lighthouse-core/audits/seo/font-size.js

```diff
@@ -265,8 +277,9 @@ class FontSize extends Audit {
       });
     }
 
+    const decimalProportion = (percentageOfPassingText / 100).toFixed(4);
```

Comment:
still a bit weird that we are passing in a string to the i18n, I would expect this to be a number, is it still necessary to call `.toFixed` now that we've sorted out the decimal piece in our i18n lib?

Reply:
> I would expect this to be a number, is it still necessary to call `.toFixed` now that we've sorted out the decimal piece in our i18n lib?

whoops, missed this, good call!

Reply:
Yup!  Good catch!